src/services/projects.js

```diff
@@ -470,131 +527,129 @@ module.exports = class ProjectsHelper {
 				['id', 'value', 'has_entities', 'validations']
 			)
 
-			// Using forEach instead of for loop for entity type validations
-
-			entityTypes.forEach((entityType) => {
-				const fieldData = projectData[entityType.value]
+			//fetch task entityType validations
+			let taskEntityTypes = await entityModelMappingQuery.findEntityTypesAndEntities(
+				{
+					model: common.TASKS,
+					status: common.STATUS_ACTIVE,
+				},
+				userDetails.organization_id,
```

Comment:
@priyanka-TL I think we should use should user resource org id , what if a user is move to an another org but he can work. on the resource he has created in previous org right?